README.md

```diff
@@ -9,6 +9,7 @@ a Rust crate for developers ~~and bindings for C and Python~~ (yet to come).
 - [Installation](#installation)
   - [Binaries](#binaries)
   - [Cargo](#cargo)
+  - [Linux packages](#linux)
```

Comment:
Please match the real title, so just "Linux", or change the real title to "Linux packages", which is not a bad idea :)